generate-stackbrew-library.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -eu
 
 declare -A aliases=(
```

Comment:
Lines 12-16 should be updated to the newer loop setup. The `update.sh` jenkins job for "repos with jq templates" does the build test one version at a time by generating a single version library file: [`./generate-stackbrew-library.sh "$version"`](https://github.com/docker-library/oi-janky-groovy/blob/5552751cc74f06d3c82252dc1d77a8f43eafe674/update.sh/versions-pipeline.groovy#L177).

```bash
if [ "$#" -eq 0 ]; then
	versions="$(jq -r 'keys | map(@sh) | join(" ")' versions.json)"
	eval "set -- $versions"
fi
```